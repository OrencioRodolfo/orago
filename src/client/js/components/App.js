import React from 'react';
import styled from 'styled-components';

const H1 = styled.div`
  color: red;
`;

const App = () => (
  <div style={{ textAlign: 'center' }}>
    <H1>Hello World</H1>
  </div>
);

export default App;
