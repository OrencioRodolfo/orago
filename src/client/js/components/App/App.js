import React from 'react';
import styled, { ThemeProvider } from 'styled-components';
import Button from '../Button/Button';
import PillLeft from '../Button/PillLeft/PillLeft';
import PillRight from '../Button/PillRight/PillRight';
import I from '../Icon/Icon';
import * as theme from '../../theme';

const H1 = styled.div`
  color: red;
`;
const AppContainer = styled.div`
  color: red;
`;

const App = () => (
  <ThemeProvider theme={theme}>
    <AppContainer style={{ textAlign: 'center' }}>
      <H1>Hello World</H1>
      <Button>
        <PillLeft>
          <I icon="star" /> Star
        </PillLeft>
        <PillRight>0</PillRight>
      </Button>
    </AppContainer>
  </ThemeProvider>
);

export default App;
