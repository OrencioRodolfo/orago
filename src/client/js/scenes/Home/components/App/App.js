import React from 'react';
import styled from 'styled-components';
import Button from '../../../../components/Button/Button';
import PillLeft from '../../../../components/Button/PillLeft/PillLeft';
import PillRight from '../../../../components/Button/PillRight/PillRight';

const H1 = styled.div`
  color: red;
`;

const App = () => (
  <div style={{ textAlign: 'center' }}>
    <H1>Hello World</H1>
    <Button>Hi</Button>
    <Button>
      <PillLeft>left content</PillLeft>
      <PillRight>right</PillRight>
    </Button>
  </div>
);

export default App;
