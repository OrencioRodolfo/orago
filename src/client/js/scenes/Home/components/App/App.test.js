import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('Must render properly', () => {
    const container = shallow(
      <App />
    );

    expect(container).toMatchSnapshot();
  });
});
