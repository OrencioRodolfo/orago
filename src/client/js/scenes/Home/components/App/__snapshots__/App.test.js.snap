// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a label 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <styled.div>
          Hello World
        </styled.div>,
        <PillLeft>
          a
        </PillLeft>,
        <PillRight>
          b
        </PillRight>,
      ],
      "style": Object {
        "textAlign": "center",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": "Hello World",
        },
        "ref": null,
        "rendered": "Hello World",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": "a",
        },
        "ref": null,
        "rendered": "a",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "function",
        "props": Object {
          "children": "b",
        },
        "ref": null,
        "rendered": "b",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <styled.div>
            Hello World
          </styled.div>,
          <PillLeft>
            a
          </PillLeft>,
          <PillRight>
            b
          </PillRight>,
        ],
        "style": Object {
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": "Hello World",
          },
          "ref": null,
          "rendered": "Hello World",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": "a",
          },
          "ref": null,
          "rendered": "a",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "function",
          "props": Object {
            "children": "b",
          },
          "ref": null,
          "rendered": "b",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
